src/meshlink.c

```diff
@@ -216,23 +215,27 @@ struct socket_in_netns_params {
 	int fd;
 };
 
+#ifdef CLONE_NEWNET
```

Comment:
I thought this would work, but it seems at least on Android NDK r19c, `CLONE_NEWNET` is defined in some header file, but `setns()` is not. So we do need to add `setns` to `AC_CHECK_FUNCS` in `configure.ac`, and replace `#ifdef CLONE_NEWNET` with `#ifdef HAVE_SETNS`.